api/src/com/cloud/network/ssl/CertService.java

```diff
@@ -14,8 +14,9 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package com.cloud.network.lb;
+package com.cloud.network.ssl;
 
```

Comment:
just a note, SSL is outdated, the official name would be TLS, name the package tls?